WebRoot/css/zm.css

```diff
@@ -3757,7 +3792,11 @@ HTML>BODY .appt-selected .appt_allday_body
 
 /* Table for a section of prefs (head + body) */
 .ZOptionsSectionTable {
-	margin-bottom:1.5em;
+	@PrefSectionContentTable@
+}
+
+.ZOptionsSectionTable * {
```

Comment:
Yes we need `*` selector here. Because in `common.css` we do have following rules and needs to override it.

```
P, TH, TD, DIV, SELECT, INPUT[type=text], INPUT[type=password], INPUT[type=file], TEXTAREA, BUTTON {
	@FontSize-normal@
}
```